### ISSUE TITLE
[BUG?] clear_screen_enter_new_location_enable doesn't seem to work as intended

### ISSUE THREAD

**Issue by nww02** [ainslec/adventuron-issue-tracker#416]
I'm not 100% sure how to use this option, but from the name is looks like the screen can be set to not clear when entering a new location (useful for games which have a sticky image at the top, and also scrollable sticky text at the bottom, such as L9 and Magnetic Scrolls).    

  The game *seems* to ignore this setting. Only setting "clear_screen_enable" being set to false fully disables the clearing of the screen on a new location redescribe, but that then causes all kinds of other problems when you *do* want to clear the screen, such as when you present a menu in on_startup, and then want the screen to clear to start the game.  This can be (mostly) circumvented using cunning (ab)use of themes, but you then end up with a problem where the page gets horribly mangled when you reload a previous save.

**Comment by ainslec** [comment]
Press F5 and try "clear_screen_enter_new_location_enable" again.

**Comment by nww02** [comment]
Hmm. I set it to "False", and allowed the default parser to handle movement, but the screen still blanks on a location-move. 🙁


________________________________

**Comment by ainslec** [comment]
Try again.

**Comment by nww02** [comment]
Yes that almost works!! But it's not "LOCK"ing the graphic at the top with the layout set to G LOCK D   .  The graphic scrolls with the text.

**Comment by ainslec** [comment]
Try again.

**Comment by ainslec** [comment]
Can you confirm this is working now?

**Comment by nww02** [comment]
I think​ so...   I'm in the middle of converting all my code back to using the standard handlers, now that this might be working...

I'll give a definitive update once I've got this avoiding all my overrides.


________________________________

**Comment by ainslec** [comment]
Is this fixed, or still an issue?
